app/scripts/controllers/app-state.js

```diff
@@ -401,6 +405,29 @@ export default class AppStateController extends EventEmitter {
     this.store.updateState({ showAccountBanner });
   }
 
+  /**
+   * Sets an object with networkName and appName
+   * or `null` if the message is meant to be cleared
+   *
+   * @param switchedNetworkDetails
```

Comment:
I see that you updated the shape of this data, but it's still documented incorrectly. Here's an attempt at rewriting it (I'm writing this on GitHub so you might want to add an extra line break or run the linter on this)

```suggestion
   * Track which network MetaMask has just automatically switched to, or call this with `null` to clear that state.
   *
   * @param {{ origin: string, networkClientId: string } | null} switchedNetworkDetails - Details about the network that MetaMask just switched to.
```

Reply:
I recall that you committed this suggestion at some point, but it's gone now :thinking:

Reply:
Restored!